css/style.css

```diff
@@ -16,4 +15,125 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+header {
+  bottom: 20px;
+  top: 40px;
+}
+
+nav {
+  margin-top: 0;
+  position: relative;
+  z-index: 5;
+  height: 71;
+  bottom: 107;
+}
+
+ul {
+  display: flex;
+  list-style: none;
+  text-align: right;
+  justify-content: flex-end;
+  padding: 0px 50px;
+  color: gray;
+  position: relative;
+  z-index: 0;
+  margin-top: -54px;
+}
+
+li {
+  padding: 20px;
+  margin-top: 120px;
+}
+
+li:hover {
+  color: rgba(224, 100, 58, 255);
+}
+
+img {
+  height: 40px;
+  top: 9px;
```

Comment:
@MonMon20  please add a wrapper (div) for your img tags and give that div a class with width:100% and give your img tags a class with width:40%